package.json

```diff
@@ -123,7 +123,7 @@
     "rollup-plugin-size-snapshot": "^0.10.0",
     "rollup-plugin-terser": "^5.1.1",
     "sinon": "^7.0.0",
-    "size-limit": "^0.21.0",
+    "size-limit": "^2.0.0",
```

Comment:
What insights did you gain over https://github.com/mui-org/material-ui/pull/16975?